Lib/test/test_turtle.py

```diff
@@ -481,11 +481,8 @@ def test_save_raises_if_parent_not_found(self) -> None:
 
         with tempfile.TemporaryDirectory() as tmpdir:
             parent = os.path.join(tmpdir, "unknown_parent")
-            msg = (
-                f"The directory '{parent}' does not exist. Cannot save to it"
-            ).replace("\\", "\\"*2)  # Escape backslashes in Windows paths
-
-            with self.assertRaisesRegex(FileNotFoundError, msg):
+            msg = f"The directory '{parent}' does not exist. Cannot save to it"
+            with self.assertRaises(FileNotFoundError, msg=msg):
```

Comment:
We attempted to do it this way first, but the tests passed even with the wrong error message, which led us to `assertRaisesRegex`.

We just checked again and changing the `msg`-variable to something different still made the tests pass, but using assertRaisesRegex made them fail.

Reply:
Ah you're right, sorry about that, I got confused. I've switched it back to escaping and confirm it fails with different exception text.